src/protocol/resp/src/request/mod.rs

```diff
@@ -36,6 +38,11 @@ pub use hmget::*;
 pub use hset::*;
 pub use hvals::*;
 pub use set::*;
+// response codes for klog
```

Comment:
I used the same codes as memcached but I'll defer to you if you'd prefer something more redis-y